src/DiseaseParm.cpp

```diff
@@ -33,6 +33,9 @@ void DiseaseParm::Initialize ()
     xmit_nc_child[2] = xmit_nc_child[3] = xmit_nc_child[4] = .08_rt*pHC;
 
     xmit_work = 0.115_rt*pWO;
+    xmit_work_diff_age = 0.090_rt*pWO;
+    xmit_work_diff_group = 0.075_rt*pWO;
+    xmit_work_diff_group_diff_age = 0.060_rt*pWO;  // later model, when a_j is older than a_i
```

Comment:
Do these lines belong to this PR? Or is this from the work you are doing with improving workplace interactions?

Reply:
true. I will remove them